modules/logging/jvm/src/main/scala/free/log4s.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.free
```

Comment:
I am not much in favor of having multiple `package` declarations. IMHO, an `import` should be as explicit and restricted as reasonably possible. In this case, it is not needed.

Reply:
Makes sense. Changed.